jp/components/grid_column_resizing.md

```diff
@@ -5,9 +5,23 @@ _keywords: Ignite UI for Angular, UI コントロール, Angular ウィジェッ
 _language: ja
 ---
 
-### グリッドの列サイズ変更
+## グリッドの列サイズ変更
 
-**列のサイズ変更**も列ごとのレベルで有効化されます。つまり、**igx-grid** にサイズ変更可能な列およびサイズ変更可能ではない列の両方を含むことが可能です。`igx-column` の `resizable` 入力によって制御されます。**igx-grid** は遅延列のサイズ変更をサポートします。つまり、列はすぐにサイズ変更されません。サイズ変更インジケーターがドラッグ操作で一時的に表示され、ドラッグ操作が終了したときに新しい列サイズが適用されます。特定の列のサイズ変更はグリッド全体の幅に影響するため、残りの列の現在の幅は変更されません。
+<p class="highlight">Column resizing feature allows the user to change the width of a certain column at runtime. The `igx-grid` control supports **deferred column resizing**. Clicking on the right header border displays a temporary resize indicator that can be dragged left or right, serving as a preview. The column does not change its width while dragging the resize indicator. The new size is applied only when the drag operation is over. Resizing a certain column affects the overall width of the grid, so the rest of the columns will preserve their current widths.</p>
```

Comment:
@ig-ealbert This section is not translated. Is this on purpose?

Reply:
@mpavlinov Yes, I wrote that in my pull request description.
"Updated resizing topic and added links (I know there is some English in here)"
Now there are conflicts, so I will try to resolve them.

Reply:
@ig-ealbert: Apologies. My bad. Didn't read the description :( .